app/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 22
+    compileSdkVersion 23
```

Comment:
I was told that compiling against version 23 could possibly get rid of the bug I was talking about, It didn't. I forgot to take this out.